.github/workflows/build.yml

```diff
@@ -60,6 +60,11 @@ jobs:
         cmake -B build
         cmake --build build --config ${{env.BUILD_TYPE}}
 
+    - name: Test
+      run: |
+        cd build/test
+        ctest
```

Comment:
```suggestion
        cmake --build test --config ${{env.BUILD_TYPE}}
```

Reply:
Are you sure it works this way? It doesn't for me locally

Reply:
It should be `cmake --build build --target=test --config ${{env.BUILD_TYPE}}` and you don't need to do `cd build/test`

Reply:
Still doesn't work on Windows. Maybe it does on Linux. I suggest you merge what we've so far - it works on all platforms now.